stream_alert/shared/resources.py

```diff
@@ -27,7 +27,7 @@ def get_required_outputs():
         set: Set of required output services and names in the form service:name
     """
     return {'{}:{}'.format(service, output)
-            for service, value in REQUIRED_OUTPUTS.iteritems()
+            for service, value in REQUIRED_OUTPUTS.items()
             for output in value.keys()}
```

Comment:
looks like this is the one failing CI now @GarretReece  - can just be ` for output in value`

Reply:
gaaaah, that's embarrassing 🤦‍♀️

Reply:
all good.. idk why it didn't complain about this before tbh...